projects/demo/src/modules/markup/skeleton/examples/1/index.ts

```diff
@@ -11,6 +11,8 @@ import {encapsulation} from '@demo/emulate/encapsulation';
     encapsulation,
 })
 export class TuiSkeletonExample1 {
+    radius = 10;
```

Comment:
Should probably be 0 if it's 0 by default

Reply:
fixed